@commitlint/lint/package.json

```
@@ -3,41 +3,13 @@
   "version": "8.3.5",
   "description": "Lint a string against commitlint rules",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
```

this should be `type` as this is not bundled definition
```suggestion
  "type": "lib/index.d.ts",
```